packages/sfui/frameworks/vue/components/SfGroup/SfGroup.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+
+defineProps({
+  invalid: {
+    type: Boolean,
+    default: false,
+  },
+  legend: {
+    type: String,
+    default: 'Label',
+  },
+  hintText: {
+    type: String,
+    default: 'Hint',
+  },
+  errorText: {
+    type: String,
+    default: 'Error message',
+  },
+});
+
+</script>
+
+<template>
+<fieldset>
+  <legend>{{ legend }}</legend>
+  <slot/>
```

Comment:
I know it's been prepared for checkboxes and radios, but I could use it for any field I guess. 🤔 I'm wondering if this could be a more generic component that would display errors and helper texts. I'm not sure if we should use legend or label then, or to display them conditionally.

Reply:
yes, if this should be more generic then need to check what should be conditionally added :)